THIRD_PARTY.txt

```diff
@@ -1,50 +0,0 @@
-This document contains the list of Third Party Software along with the license information.
-
-Third Party Software may impose additional restrictions and it is the user's responsibility to ensure that they have met the licensing
-requirements of the relevant license of the Third Party Software they are using.
-
-//-----//
-AndroidX
-- multidex
-- constraintlayout
-- gridlayout
-- appcompat
-- recyclerview
-- test
-- runner
-AndroidX and his submodules are distributed under the terms and conditions of the Apache 2.0 License (http://www.apache.org/licenses/LICENSE-2.0)
-
-Material
-Material is distributed under the terms and conditions of the Apache 2.0 License (http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
Should we keep Material? I would say yes because it's not part of AndroidX.

Also, should we add Google services in the list?